imap_processing/tests/idex/test_idex_l0.py

```diff
@@ -7,7 +7,7 @@
 
 
 def test_idex_decom_length(decom_test_data: xr.Dataset):
-    """Verify that there are 6 data variables in the output.
+    """Verify that there are 42 data variables in the output.
```

Comment:
```suggestion
    """Verify that the output data has the expected number of data variables.
```

Could remove these numbers here and make the wording more generic, in case the number changes again in the future. (same in the other docstring)